Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM node:carbon
 WORKDIR /src/
 COPY package*.json ./
 RUN npm install
+RUN npm install amqplib
+RUN npm install xmlhttprequest
```

Comment:
Are these packages not in package.json?

Reply:
no

Reply:
If you add them to package.json, do you still need to manually install them?